scico/ladmm.py

```diff
@@ -0,0 +1,365 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2021 by SCICO Developers
+# All rights reserved. BSD 3-clause License.
+# This file is part of the SCICO package. Details of the copyright and
+# user license can be found in the 'LICENSE' file distributed with the
+# package.
+
+"""Linearized ADMM solver."""
+
+# Needed to annotate a class method that returns the encapsulating class;
+# see https://www.python.org/dev/peps/pep-0563/
+from __future__ import annotations
+
+from typing import Callable, List, Optional, Tuple, Union
+
+import scico.numpy as snp
+from scico.blockarray import BlockArray
+from scico.diagnostics import IterationStats
+from scico.functional import Functional
+from scico.linop import LinearOperator
+from scico.numpy.linalg import norm
+from scico.typing import JaxArray
+from scico.util import Timer, ensure_on_device
+
+__author__ = """\n""".join(
+    ["Luke Pfister <luke.pfister@gmail.com>", "Brendt Wohlberg <brendt@ieee.org>"]
+)
+
+
+class LinearizedADMM:
+    r"""Linearized Alternating Direction Method of Multipliers algorithm.
```

Comment:
As per style guide: "Linearized alternating direction method of multipliers algorithm"

Reply:
Fixed.

Reply:
Similar style changes still need to be applied to existing modules, such as `scico.admm`. Can you take care of that within the `mike/docs` branch?